src/views/darkroom.c

```diff
@@ -2394,6 +2397,25 @@ void init_key_accels(dt_view_t *self)
 
   // fullscreen view
   dt_accel_register_view(self, NC_("accel", "full preview"), GDK_KEY_z, 0);
+
+  // undo/redo
+  dt_accel_register_view(self, NC_("accel", "undo"), GDK_KEY_z, GDK_CONTROL_MASK);
+  dt_accel_register_view(self, NC_("accel", "redo"), GDK_KEY_y, GDK_CONTROL_MASK);
```

Comment:
You mean as default? I don't have a strong opinion, this can always be changed in the preference. It is indeed the case that ctrl-y or ctrl--shif-z are both used. GIMP is using ctrl-y that's why I've used this.